README.md

```diff
@@ -94,7 +93,6 @@ We also have variants that are designed to be Kubernetes worker nodes in VMware:
 
 The following variants are designed to be Kubernetes worker nodes on bare metal:
 
-* `metal-k8s-1.24`
```

Comment:
Do we want to add a bullet point to "The following variants are no longer supported:" below?

```
* Bare metal and VMware variants using Kubernetes 1.24 and earlier
```